src/Conf.ml

```diff
@@ -15,6 +15,7 @@ type t =
   { break_cases: [`Fit | `Nested | `All]
   ; break_collection_expressions: [`Wrap | `Fit_or_vertical]
   ; break_infix: [`Wrap | `Fit_or_vertical]
+  ; break_separators: [`Before | `After]
```

Comment:
Your choice, but my impression from reading the code is that it would be simpler if this was a bool treated as `break_struct` is.

Reply:
I don't know, I'm not a fan of this choice for `break_struct` to begin with, I find the variants to be clearer than the boolean

Reply:
Ok, I don't have a very strong position, but let me explain my thinking to make sure we're on the same page. The trade-off is between the more immediately clear semantics of the variants on the one hand versus the less disruptive code to branch on booleans vs variants. In cases where there are multiple points in the core formatting code that branch on an option, I find that it is easier to keep in mind how they fit into the surrounding formatting code when they are booleans. It does mean that you need to have in mind what the booleans mean, but with careful naming this seems to be not a huge issue. I agree that it is more immediately clear what the meaning is with variants, but basically I would rather optimize for the formatting code, esp. when there are multiple occurrences far apart.